.github/workflows/app.yml

```diff
@@ -0,0 +1,36 @@
+name: Test and build Jellyfin Android
+on:
+  push:
+  release:
+    types: [ published ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
I'd prefer to pin it to a version because Microsoft will definitely break the image like they always do. In addition, we should use a step to install JDK 11 or higher instead of the default JDK from the image.

https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-java-with-gradle#specifying-the-jvm-version-and-architecture